generators/server/templates/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,110 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
```

Comment:
you need to add this file to the server files section so that it is copied when generating anew project. but do we really need this file since the wrapper binary is included already? @PierreBesson WDYT

Reply:
I not sure this part.  It just provide a new way to download the jar file. 
Try wget , try curl , try this downloaded. 
Yeah, we have the jar file already, this may not needed. But I thinks it would be better to keep the original logic of the maven-wrapper. WDUT?